### PR TITLE
ajouter mon projet open source netflixCollection

### PR DESCRIPTION
**Pull request by mouhamedhanne**
NetflixCollection is your perfect companion for easy management of your favorite movies and TV shows. Create your own personalized collection by adding direct links to your favorite films on Netflix, Disney+, Amazon Prime, and other streaming platforms. Discover the world of cinema in a smarter and more organized way with NetflixCollection.

### CODE REVIEW

**Review by ManuSquall — CHANGES_REQUESTED** [review]
Please check the [contributing guidelines](https://github.com/GalsenDev221/made.in.senegal/blob/master/CONTRIBUTING.md) 🙃

Will be glad to merge it once good

File: README.md [hunk 1]
```diff
@@ -152,7 +152,10 @@ If you are the maintainer of a project, you can add our badge by copying the cod
 
 ## <a name="N"> </a>N
 
-- **[NestJs MongoDB Base Crud API](https://github.com/babacarbasse/nestjs-mongo-crud-base)** - A NestJS package that allow you setup quickly CRUD entities in your NestJS Mongo Application. by **[@babacarbasse](https://github.com/babacarbasse)**
+- **[NestJs MongoDB Base Crud API](https://github.com/babacarbasse/nestjs-mongo-crud-base) by **[Geek Genius](https://github.com/mouhamedhanne)\*\*
```
Comment (daoodaba975) [inline README.md:155]: Hi @mouhamedhanne need some change, do not change data already on file

File: README.md [hunk 2]
```diff
@@ -152,7 +152,10 @@ If you are the maintainer of a project, you can add our badge by copying the cod
...
+- **[NumMenu Bot](https://github.com/DerXter/NumMenu-Bot)** -NetflixCollection is your perfect companion for easy management of your favorite movies and TV shows. Create your own personalized collection by adding direct links to your favorite films on Netflix, Disney+, Amazon Prime, and other streaming platforms. Discover the world of cinema in a smarter and more organized way with NetflixCollection. by **[@DerXter](https://github.com/DerXter)**
+
+- **[NetflixCollection](https://github.com/mouhamedhanne/Netflix_collection)** - A NestJS package that allow you setup quickly CRUD entities in your NestJS Mongo Application. by **[@babacarbasse](https://github.com/babacarbasse)**
```
Comment (daoodaba975) [inline README.md:158]: Please check [contributing guide](https://github.com/GalsenDev221/made.in.senegal/blob/master/CONTRIBUTING.md) it will help you!

**Review by daoodaba975 — CHANGES_REQUESTED** [review]

**Review by daoodaba975 — APPROVED** [review]

**Comment by daoodaba975** [comment]
@mouhamedhanne Next time take time to view the contributing guide.

**Review by ManuSquall — APPROVED** [review]
LGTM 🥳!

**Pull request merged**